src/Sentry/SentryFailedRequestHandler.cs

```diff
@@ -0,0 +1,100 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Sentry.Protocol;
+
+namespace Sentry
+{
+    internal class SentryFailedRequestHandler : ISentryFailedRequestHandler
+    {
+        private readonly IHub _hub;
+        private readonly SentryOptions? _options;
+
+        public string MechanismType { get => "SentryFailedRequestHandler"; }
+
+        /// <summary>
+        /// Initializes an instance of <see cref="SentryFailedRequestHandler"/>.
+        /// </summary>
```

Comment:
We only require XML summary comments for public members.  I'd omit them if they don't add any useful information.

Reply:
Sounds good. I think VS generated those comments automatically when I added the class... but I'll strip them out.